client/src/Header.js

```diff
@@ -0,0 +1,47 @@
+import './App.css';
+import {useState} from 'react'
+import {BsFillCartFill} from 'react-icons/bs'
+import { BsFillSuitHeartFill } from 'react-icons/bs'
+import { ImSearch } from 'react-icons/im'
+
+export default function Header() {
+  const [search, setSearch] = useState('')
+
+  const handleChange = (event) => {
+    setSearch(event.target.value);
+  }
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    try {
+      const response = await fetch(`https://www.googleapis.com/books/v1/volumes?q=${search}&key=AIzaSyCQ7EBy8yMIYuOdh350WfDgEwWKGJSWAh4`)
+      const jsonData= await response.json();
```

Comment:
It's best to check `ok` before calling `response.json()` because if the response is not `ok`, there might not be valid JSON in the response. That would cause this call to throw, and you wouldn't get a chance to display an error.